src/shared-styles/_rem.scss

```diff
@@ -1,9 +1,14 @@
 @use "font";
 @use "vendor/sass-rem/rem" as sass-rem with (
   $rem-baseline: font.$size,
-  $rem-fallback: false,
 );
 
+@forward "vendor/sass-rem/rem";
```

Comment:
What's this for?

Reply:
Left over code, not needed now we're wrapping what we use from `sass-rem`. Will remove.